io.github.jordanl2.ModularCalculator.yml

```diff
@@ -0,0 +1,30 @@
+app-id: io.github.jordanl2.ModularCalculator
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+default-branch: stable
+command: modularcalculator
```

Comment:
this is probably missing `rename-destkop-file`

Reply:
No, modularcalculator is a python script entry point.

Reply:
I didn't say to replace the line. but you'll see on the build failure.